docs/scripts/phenomic.node.js

```diff
@@ -0,0 +1,13 @@
+import metadata from "../web_modules/app/metadata"
+import routes from "../web_modules/app/routes"
+import store from "../web_modules/app/store"
+
+import phenomicStatic from "phenomic/lib/static"
+
+module.exports = (options) =>
```

Comment:
node compat. This script is consumed via node. But I will try to use export default and adjust in the core usage